bootloader/bootloadertest/bootloadertest.go

```diff
@@ -453,6 +454,25 @@ func (b *MockTrustedAssetsBootloader) TrustedAssets() ([]string, error) {
 	return b.TrustedAssetsList, b.TrustedAssetsErr
 }
 
+func (b *MockTrustedAssetsBootloader) RecoveryBootChain(kernelPath string) ([]bootloader.BootFile, error) {
+	chain := []bootloader.BootFile{
```

Comment:
this is unsual, most mock methods use mock values attached to the mocked bootloader, not a fixed set of values

Reply:
Yes, I did this as a quick placeholder last night to be able to run the tests. I'm currently updating it to produce a result consistent with the mocked trusted assets.

Reply:
Returning mock values attached to the mocked bootloader seem the best approach in this case, will reimplement as that.